pulse_actions/worker.py

```diff
@@ -3,6 +3,8 @@
 import os
 import traceback
 
+from authentication import get_user_and_password
```

Comment:
Can you change this to an absolute import (from pulse_actions.authentication import get_user_and_password)?